### PR TITLE
Upgrade deps

### PR DESCRIPTION
**Pull request by sujitn**
.netstandard 2.0
upgrade protobuf 3.6.1

### CODE REVIEW

File: STAN.CLIENT/Protocol.cs [hunk 1]
```diff
@@ -1,5 +1,7 @@
-// Generated by the protocol buffer compiler.  DO NOT EDIT!
-// source: protocol.proto
+// <auto-generated>
```
Comment (watfordgnf) [inline STAN.CLIENT/Protocol.cs]: This includes the PING/PONG support still.

File: STAN.CLIENT/STAN.CLIENT.csproj [hunk 2]
```diff
@@ -1,15 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
```
Comment (watfordgnf) [inline STAN.CLIENT/STAN.CLIENT.csproj]: Could you clean up the extraneous newlines in this file?

File: STAN.CLIENT/pb/protocol.proto [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2015-2018 The NATS Authors
+// Copyright 2016-2018 The NATS Authors
```
Comment (watfordgnf) [inline STAN.CLIENT/pb/protocol.proto]: This should be reverted back to the copy on master.

Comment (sujitn) [reply STAN.CLIENT/pb/protocol.proto]: reverted to old proto

File: STAN.CLIENT/Protocol.cs (hunk 1 above)
Comment (sujitn) [reply STAN.CLIENT/Protocol.cs]: re-generated using old proto file

**Comment by sujitn** [comment]
Reverted all ping proto references, hopefully everything should in order now. Thanks

**Comment by sujitn** [comment]
closing this. "pings" merge seemed to contain the upgrade protobuf dependencies

**Pull request closed without merging**